bot/exts/utilities/bookmark.py

```diff
@@ -152,6 +204,27 @@ def event_check(reaction: discord.Reaction, user: discord.Member) -> bool:
 
         await reaction_message.delete()
 
+    @commands.dm_only()
```

Comment:
I understand. I don't think the UI would make too much sense if a bookmark cannot be deleted from dms.

I feel like it would make the most sense to take a reply or a message as a parameter. if a user uses `.bm del` and replying to a bookmark, it should delete it, just like `.bm` can be used to bookmark a reply.